internal/sdk/cloudian/sdk.go

```diff
@@ -112,6 +112,30 @@ type SecurityInfo struct {
 	SecretKey string `json:"secretKey"`
 }
 
+// QoS is the Cloudian API's term for limits on quotas, counts and rates enforced on a `User`
+type QoS struct {
+	// Max storage quota
+	StorageQuota ByteSize
+	// Warning limit storage quota
+	StorageQuotaWarning ByteSize
+	// Max storage quota in number of objects
+	StorageQuotaCount uint64
```

Comment:
To disable the limits, we need to support `-1`

```suggestion
	StorageQuotaCount int64
```

Reply:
Could also make them pointer and transform nil to -1 and positive values to strings. Zero value would then be "unlimited", which is kinda nice